webapp/actions/global_actions.jsx

```diff
@@ -534,3 +538,33 @@ export function emitBrowserFocus(focus) {
         focus
     });
 }
+
+export function redirectUserToDefaultTeam() {
+    const teams = TeamStore.getAll();
+    const teamMembers = TeamStore.getMyTeamMembers();
+    let teamId = BrowserStore.getGlobalItem('team');
```

Comment:
Instead of storing this locally, this could be saved in prefs and be accessible across devices. I think that's how we do last channel viewed